src/features/wallet/root/ActivityCard.tsx

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react'
+import { Animated, TouchableOpacity } from 'react-native'
+import SlidingUpPanel from 'rn-sliding-up-panel'
+import { orderBy, random, times } from 'lodash'
+import Haptic from 'react-native-haptic-feedback'
+import CardHandle from './CardHandle'
+import Box from '../../../components/Box'
+import Text from '../../../components/Text'
+import ActivityItem from './ActivityItem'
+import CarotDown from '../../../assets/images/carot-down.svg'
+
+type Props = {
+  animatedValue: Animated.Value
+}
+
+const ActivityCard = ({ animatedValue }: Props) => {
+  const renderItem = ({ item, index }) => (
+    <ActivityItem
+      type={item.type}
+      time={item.time}
+      amount={item.amount}
+      isFirst={index === 0}
+      isLast={index === data.length - 1}
+    />
+  )
+
+  const handlePress = () => {
+    Haptic.trigger('notificationWarning')
+  }
+
+  return (
+    <SlidingUpPanel
```

Comment:
FYI I think I'm going to end up removing this in favor of a custom pan responder in order to get the springy effect